package.json

```diff
@@ -6,15 +6,18 @@
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "scripts": {
-    "lint": "eslint . --ext .js,.html",
+    "lint": "eslint .",
     "test": "npm run lint && npm run test:unit",
     "test:unit": "d2l-test-runner"
   },
   "devDependencies": {
     "@brightspace-ui/testing": "^1",
     "@web/dev-server": "^0.4",
-    "eslint": "^8",
-    "eslint-config-brightspace": "^1",
+    "eslint": "^9",
+    "eslint-config-brightspace": "^1.2.1",
```

Comment:
Do we need the minor and patch here?

Reply:
Yes, anything under 1.2.1 adds support for the html plugin that works with eslint9